ios/Podfile

```diff
@@ -5,6 +5,8 @@ target 'emurgo' do
   # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
   # use_frameworks!
 
+  pod 'react-native-chain-libs', :path => '../node_modules/react-native-chain-libs'
```

Comment:
this shouldn’t work because it uses React as a dependency so it’s going to install React again using Cocopoads. The library should be added manually.